### ISSUE TITLE
Date casting

### ISSUE THREAD

**Issue by edwinvdpol** [Soyhuce/next-ide-helper#47]
Hi there, it's me again ;-)

I noticed that when using a date cast like below, it does not parse this as a date instance:

```php
<?php

class Test extends Model
{
    protected $casts = [
        'first' => 'date:Y-m-d',
        'second' => 'datetime:Y-m-d H:i:s',
    ];
}
```

Is this by design?

It is allowed by Laravel, see the [date casting documentation](https://laravel.com/docs/8.x/eloquent-mutators#date-casting).

PS: I am really starting to like this package. More clean and correct than IDE helper by Barry. 👍

**Comment by bastien-phi** [comment]
I never used it so I did not implement this behaviour. I will look this.

Some Laravel features are not implemented in this package because I never needed them. Feel free to open issues, I will do my best (between client works). PR are also welcomed !

Glad you like it :smiley:

**Comment by bastien-phi** [comment]
Released as 0.7.2

**Comment by edwinvdpol** [comment]
Thank you!
